pkg/osquery/extension.go

```diff
@@ -355,15 +373,14 @@ func (e *Extension) generateConfigsWithReenroll(ctx context.Context, reenroll bo
 			return "", errors.Wrap(err, "enrollment invalid, reenrollment errored")
 		}
 		if invalid {
-			return "", errors.New("enrollment invalid, reenrollment invalid")
+			return "", reenrollmentInvalidErr
```

Comment:
since you only use it in here, i would stick with the string and pull the comment inside, instead of that var

Reply:
I pulled it out so it could be checked. The todo refers to the checking once we decide how to handle that.